DotnetRuntimeMetadata.json

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
@adityapatwardhan, your last commit had 1 failures in `PowerShell-CI-macos`
Test-Connection.TraceRoute.TraceRoute works
```powershell
Expected strings to be the same, but they were different.
String lengths are both 14.
Strings differ at index 5.
Expected: 'Mac-1373.local'
But was:  'Mac-1420.local'
at <ScriptBlock>, /Users/runner/runners/2.166.2/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1: line 257
257:             $result[0].Source | Should -BeExactly $hostName
```